qnl/key-routing-service/src/main/java/com/uwaterloo/iqc/qnl/qll/cqptoolkit/server/KeyTransferServer.java

```diff
@@ -118,5 +117,17 @@ public void onKeyFromCQP(Key keyMessage, StreamObserver<Empty> responseObserver)
                 keyListener.onKeyGenerated();
             }
         }
+
+        @Override
+        public void onKeyFromCQP(Key keyMessage, StreamObserver<Empty> responseObserver) {
+            LOGGER.info("Received key from CQP.");
```

Comment:
Remove the log line. Or you can change the log level to DEBUG.

Reply:
Changed line to `debug`